uniconnect-epfl/uniconnect
# fix: solve layout and announcement problems

# What I did
The app notifies the user when he has no events yet. 
I added the sticky headers for the events screen so we know if we are seeing upcoming events or past ones.
We can now see the selected interests when we create an announcement

# How I did it
I added an empty array for events in the creation method, so the query in the database is not throwing an error but retrieve an empty array. It's detected and a different message than "Error with your connection is shown". 

# How to verify it
You can check the event screen to see the sticky header. 
You can try to go to the event screen without any events to see the correct notification "No events yet"
When selecting the interests for an announcement, you can now see what you selected.
# Demo video
<img width="377" alt="image" src="https://github.com/uniconnect-epfl/uniconnect/assets/108724823/2ad1497e-f3b2-4d4c-8740-9ff0165b9d46">
<img width="387" alt="image" src="https://github.com/uniconnect-epfl/uniconnect/assets/108724823/c0edcda3-1c61-461c-a26d-d1b58af385f9">
<img width="381" alt="image" src="https://github.com/uniconnect-epfl/uniconnect/assets/108724823/f09d1c7e-964e-4e41-a7e7-865a74f71e1d">


# Pre-merge checklist
The changes I have introduced:
